test/unit/models/bxml/test_phone_number.py

```diff
@@ -27,3 +27,8 @@ def test_instance(self):
     def test_to_bxml(self):
         expected = '<PhoneNumber transferAnswerUrl="https://example.com/webhooks/transfer_answer" transferAnswerMethod="POST" tag="">+19195551234</PhoneNumber>'
         assert expected == self.phone_number.to_bxml()
+
```

Comment:
you can just add uui to the instance created in `seUp` no need to make another test function for this